ui/app/AppLayouts/Chat/views/ContactsColumnView.qml

```diff
@@ -21,7 +21,7 @@ import "../panels"
 import "../popups"
 import AppLayouts.Chat.stores 1.0
 import AppLayouts.Communities.popups 1.0
-import AppLayouts.Profile.stores 1.0
+import AppLayouts.stores 1.0 as AppLayoutStores
```

Comment:
Is this import even needed then? There's no other change in this file that would reference it